bin/build_project.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+export WORKDIR=`pwd`
+echo "(RE)Building Project ..."
+
+echo "Activating virtual enviroment..."
+source ../env/bin/activate
+
+echo "Installing Python Dependencies..."
+pip install -r requirements.txt
+echo
+CODE=$?
+[ $CODE -ne 0 ] && exit $CODE
+
+echo "Updating NPM packages..."
+npm install
+echo
+CODE=$?
+[ $CODE -ne 0 ] && exit $CODE
+
+echo "Building JS ..."
```

Comment:
I suggest to rename message to
`Building Webpack Dev build ...`
because `build_js` with flags, for example `-p` will build Prod minified bundle.